augur/curate/__init__.py

```diff
@@ -138,6 +138,9 @@ def run(args):
     if args.metadata == '-':
         args.metadata = sys.stdin
 
+    if args.metadata_delimiters == DEFER_LIST_DEFAULT:
```

Comment:
Usually sentinel values like this are compared for exact object identity, e.g. `… is DEFER_LIST_DEFAULT`, instead of equality (`==`).

Reply:
Updated in [force-push](https://github.com/nextstrain/augur/compare/4a25d3bdfa3ec9a3cbd98f021314e0ca6579bef7..20206183a811c37e6d22fc3da9d57113fc8ee186)